### PR TITLE
Sync Canvas metrics w/ Details Panel

### PR DESCRIPTION
**Pull request by foot**
Address #1278

### CODE REVIEW

**Comment by davkal** [comment]
Brainstorm:

* even though details and canvas metrics are are requested at different times, the samples' dates should be the same, there is potential to sync them
* maybe remove metrics feeder HOC, to get the sync right between details and canvas, then re-add feeder
* central metrics object for that stores metrics by node id, can be queried with a timestamp, feeder HOC feeds that timestamp
* global metrics timestamp clock, dynamically adjusted based on what sample ranges are available, keeps all metrics in sync

**Comment by foot** [comment]
Mmm, I like the idea of global metrics timestamp clock!

File: client/app/scripts/charts/nodes-chart-nodes.js [hunk 1]
```diff
@@ -32,11 +33,13 @@ export default class NodesChartNodes extends React.Component {
     };
 
     // TODO: think about pulling this up into the store.
-    const metric = node => (
-      node.get('metrics') && node.get('metrics')
+    const metric = node => {
+      const isHighlighted = topCardNode && topCardNode.details && topCardNode.id === node.get('id');
+      const sourceNode = isHighlighted ? fromJS(topCardNode.details) : node;
```
Comment (davkal) [inline client/app/scripts/charts/nodes-chart-nodes.js]: Sneaky 🐍

**Comment by davkal** [comment]
LGTM

**Pull request closed without merging**